cli/src/config.rs

```diff
@@ -1,6 +1,8 @@
 use std::env;
-use std::fs;
+use std::fs::{self, Permissions};
```

Comment:
Somewhat nitpicky, but this will warn on Windows as an unused import.

```suggestion
use std::fs;
#[cfg(unix)]
use std::fs::Permissions;
```

Reply:
Great point.